src/bci_build/package/samba-toolbox/README.md.j2

```diff
@@ -0,0 +1,19 @@
+# {{ image.pretty_name }} Container Image
+
+{% include 'badges.j2' %}
+
+## Description
+
+Samba is a feature-rich Open Source implementation of the SMB and Active Directory protocols for Linux and UNIX-like systems.
+
+This image comes with Samba tools and TDB tools.
+
+## Usage
+
+To connect to a SMB file server, run the following command:
```

Comment:
maybe use a different example for toolbox as that seems to be the same for client. is there anything else in toolbox?

Reply:
There's not much that the toolbox can do, but I updated a bit to include usage of TDB tools as well.